analysis_runner/__init__.py

```diff
@@ -38,12 +42,51 @@ def output_path(filename: str, bucket_category: Optional[str] = None) -> str:
     """
     dataset = os.getenv('DATASET')
     access_level = os.getenv('ACCESS_LEVEL')
-    output = os.getenv('OUTPUT')
-    assert dataset and access_level and output
+    assert dataset and access_level
 
     if bucket_category is None:
         bucket_category = 'test' if access_level == 'test' else 'main'
     elif bucket_category not in ('archive', 'upload'):
         bucket_category = f'{access_level}-{bucket_category}'
 
-    return os.path.join('gs://', f'cpg-{dataset}-{bucket_category}', output, filename)
+    return os.path.join('gs://', f'cpg-{dataset}-{bucket_category}', path)
+
+
+def output_path(path_suffix: str, bucket_category: Optional[str] = None) -> str:
+    """Returns a full GCS path for the given bucket category and path suffix.
+
+    In contrast to the bucket_path function, output_path takes the "output" parameter
+    from the analysis-runner invocation into account.
+
+    Examples
+    --------
+
+    Assuming that the analysis-runner has been invoked with
+    `--dataset fewgenomes --access-level test --output 1kg_pca/v42`:
+
+    >>> from analysis_runner import output_path
```

Comment:
Do these examples get automatically tested?

Reply:
If not, would be cool at some point to add these doctests to the GH action.

Reply:
Well, this particular one would rely on environment variables being set. Not sure if one could automate that easily?

Reply:
Ah true, not a further consideration then :)